iati/tests/test_rulesets.py

```diff
@@ -1,7 +1,6 @@
 """A module containing tests for the library representation of Rulesets.
 
 Todo:
```

Comment:
Maybe add a thing to test for case not being able to be changed after instantiation?

Reply:
https://github.com/IATI/pyIATI/pull/252/commits/73e79a6f405b2f059ff3c62aa97c76e4df1d2bd8 cherry-picked from https://github.com/IATI/pyIATI/pull/249/commits/98342bbb3502b8218c3023e1f78ad5467a01cb38